tests/delete/models.py

```diff
@@ -241,3 +241,115 @@ class GenericDeleteBottomParent(models.Model):
     generic_delete_bottom = models.ForeignKey(
         GenericDeleteBottom, on_delete=models.CASCADE
     )
+
+
+class Foo(models.Model):
+    """Initial model named Foo"""
+
+
+class Bar(models.Model):
```

Comment:
Please try to avoid meaningless model names (e.g. `Foo`, `Bar`, `Baz`, etc.). Maybe `Child`, `Parent`, `Grandparent`, etc.